<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplaceDialog</class>
 <widget class="QDialog" name="FindReplaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout">
     <item>
      <layout class="QGridLayout" name="findGridLayout" columnstretch="0,1,1">
       <item row="0" column="0">
        <widget class="QLabel" name="findLabel">
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QLineEdit" name="findLindEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="replaceLabel">
         <property name="text">
          <string>Replace with</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="replaceLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="directonLabel">
         <property name="text">
          <string>Direction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QRadioButton" name="forwardRadioButton">
         <property name="text">
          <string>Forward</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QRadioButton" name="backwardRadioButton">
         <property name="text">
          <string>Backward</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="checkBoxesGridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="matchCaseCheckBox">
         <property name="text">
          <string>Match case</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCheckBox" name="matchWordCheckBox">
         <property name="text">
          <string>Match whole word only</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="wrapSearchCheckBox">
         <property name="text">
          <string>Wrap search</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="regularExpressionCheckBox">
         <property name="text">
          <string>Regular expression</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="bottomverticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="buttonsVerticalLayout">
     <item>
      <widget class="QPushButton" name="findPushButton">
       <property name="text">
        <string>Find</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replacePushButton">
       <property name="text">
        <string>Replace</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="replaceAllPushButton">
       <property name="text">
        <string>Replace All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="buttontsVerticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
